### ISSUE TITLE
No prebuilt binaries for node 5.0 on GNU/Linux amd64

### ISSUE THREAD

**Issue by dos1** [sass/node-sass#1245]
I'm trying to build a docker image out of official node image for a project where one of its dependencies is node-sass.

With node:4 image everything works as expected, but for node:5 it tries to compile node-sass with gyp and fails.

```
gyp verb command build []                                                                                                                                                                                                                          
gyp verb build type Release                                                                                                                                                                                                                        
gyp verb architecture x64                                                                                                                                                                                                                          
gyp verb node dev dir /usr/local/lib/node_modules/harp/node_modules/node-sass/.node-gyp/5.0.0                                                                                                                                                      
gyp verb `which` succeeded for `make` /usr/bin/make
gyp info spawn make
gyp info spawn args [ 'V=1', 'BUILDTYPE=Release', '-C', 'build' ]
make: Entering directory '/usr/local/lib/node_modules/harp/node_modules/node-sass/build'
  g++ '-DNODE_GYP_MODULE_NAME=binding' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DBUILDING_NODE_EXTENSION' -I/usr/local/lib/node_modules/harp/node_modules/node-sass/.node-gyp/5.0.0/include/node -I/usr/local/lib/node_modules/harp/node_modules/node-sass/.node-gyp/5.0.0/src -I/usr/local/lib/node_modules/harp/node_modules/node-sass/.node-gyp/5.0.0/deps/uv/include -I/usr/local/lib/node_modules/harp/node_modules/node-sass/.node-gyp/5.0.0/deps/v8/include -I../../nan  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -ffunction-sections -fdata-sections -fno-omit-frame-pointer -fno-rtti -fno-exceptions -std=gnu++0x -std=c++0x -MMD -MF ./Release/.deps/Release/obj.target/binding/src/binding.o.d.raw  -c -o Release/obj.target/binding/src/binding.o ../src/binding.cpp
In file included from ../src/binding.cpp:3:0:
../src/sass_context_wrapper.h:8:26: fatal error: sass_context.h: No such file or directory
 #include <sass_context.h>
                          ^
compilation terminated.
binding.target.mk:105: recipe for target 'Release/obj.target/binding/src/binding.o' failed
make: Leaving directory '/usr/local/lib/node_modules/harp/node_modules/node-sass/build'
make: *** [Release/obj.target/binding/src/binding.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/harp/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 4.2.0-17-generic
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/harp/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /usr/local/lib/node_modules/harp/node_modules/node-sass
gyp ERR! node -v v5.0.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok 
Build failed
npm info lifecycle node-sass@3.3.3~postinstall: Failed to exec postinstall script
npm ERR! Linux 4.2.0-17-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "harp" "-g"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE
```

I presume that the build issue itself comes from either lack of libsass-dev or too old GCC in Debian Jessie on which the node image in Docker is based. However, it should use pre-build binary bindings - seems like the one matching node 5.0's v8 is missing?

**Comment by dos1** [comment]
Okay, now I see that is tried to use older node-sass version, which is kinda reasonable that it doesn't have binaries for node 5.0.

```
Cannot download "https://github.com/sass/node-sass/releases/download/v3.3.3/linux-x64-47_binding.node": HTTP error 404 Not Found
npm info lifecycle node-sass@3.3.3~postinstall: node-sass@3.3.3
```

**Comment by xzyfer** [comment]
Correct. Node 5 support only landed in 3.4.1.
